token/token.go

```diff
@@ -600,20 +601,23 @@ func ToNumber(value string) *NumberValue {
 	}
 }
 
-func looksLikeTimeValue(value string) bool {
```

Comment:
I've looked at [the original PR](https://github.com/goccy/go-yaml/pull/158), yet it was not clear to me why we needed to treat time values differently, so I removed the logic at all, but I'm entirely sure if that was OK or not...